test/runtests.jl

```diff
@@ -26,7 +26,11 @@ cd(dirname(@__FILE__)) do
     include("tutorial_raster.jl")
     include("tutorial_vector.jl")
     include("tutorial_vrt.jl")
-    include("gdal_utils.jl")
+    # several conversions for ccall are only added in v0.5
```

Comment:
If I remove the condition the first error I get is
```MethodError: `convert` has no method matching convert(::Type{Ptr{GDAL.GDALDatasetH}}, ::Array{Ptr{GDAL.GDALDatasetH},1})```
from https://github.com/visr/GDAL.jl/blob/453ba74b8b8e4f6292a80c4f1c4128f1e7f4a280/test/gdal_utils.jl#L61

Which is a bit odd since if I test on v0.4
```
julia> abstract Foo
julia> Base.cconvert(Ptr{Foo}, [Ptr{Foo}(C_NULL)])
1-element Array{Ptr{Foo},1}:
 Ptr{Foo} @0x0000000000000000
```
it seems to work fine. Since it was working on v0.5 I didn't want to go to the bottom of it.

But the function does seem to expect a list of input datasets.
https://github.com/visr/GDAL.jl/blob/453ba74b8b8e4f6292a80c4f1c4128f1e7f4a280/src/gdal_utils.jl#L200-L201